src/main/js/components/cells/date/DateCell.jsx

```diff
@@ -7,8 +7,15 @@ import DateEditCell from "./DateEditCell";
 import TableauxConstants from "../../../constants/TableauxConstants";
 import Moment from "moment";
 import ActionCreator from "../../../actions/ActionCreator";
+import keyMirror from "keymirror";
 const DateFormats = TableauxConstants.DateFormats;
 
+const OPTIONS = keyMirror({
```

Comment:
At some point, we should check out something different than `keyMirror`. Something that takes a list and generates a map from it simply.

```
const OPTIONS = enum(["SAVE", "CANCEL", "CLEAR"]);
// would generate:
const OPTIONS = {
  "SAVE": "SAVE",
  "CANCEL": "CANCEL",
  "CLEAR": "CLEAR"
};
```